### ISSUE TITLE
Segfault in park entrance price tab

### ISSUE THREAD

**Issue by TyRoXx** [OpenRCT2/OpenRCT2#9630]
**OS:** Linux Mint 19.1 (64 bit), English
**Commit/Build:** 5733d666b374786e1317ec3b5f5f60b96d09598d (bisected)

Originally I installed the game from the Ubuntu PPA. The version information is incomplete, but I installed it on 2019-07-22.
```
$ openrct2 --version
OpenRCT2,  ( on , DEBUG)
Linux (x86-64)
```

The game crashed (see below). After that I checked out the source code and bisected the crash with git beginning with an older release. This commit seemed to introduce the crash: 5733d666b374786e1317ec3b5f5f60b96d09598d

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. start a new game (Crazy Castle). I tried both English and German. The language makes no difference
2. open Park Information
3. click the park entrance price tab
4. the game will crash with a segmentation fault

According to my debugger the invalid read happened in strlen:
```
1   __strlen_sse2                                                                                                                                                                                   strlen.S         120  0x7ffff4f83646 
2   format_append_string                                                                                                                                                                            Localisation.cpp 373  0x55555580b688 
3   format_string_code                                                                                                                                                                              Localisation.cpp 1109 0x55555580d5cb 
4   format_string_part_from_raw                                                                                                                                                                     Localisation.cpp 1228 0x55555580dc6f 
5   format_string_part                                                                                                                                                                              Localisation.cpp 1256 0x55555580dd7c 
6   format_string_code                                                                                                                                                                              Localisation.cpp 1101 0x55555580d57f 
7   format_string_part_from_raw                                                                                                                                                                     Localisation.cpp 1228 0x55555580dc6f 
8   format_string_part                                                                                                                                                                              Localisation.cpp 1256 0x55555580dd7c 
9   format_string                                                                                                                                                                                   Localisation.cpp 1343 0x55555580e1e3 
10  DrawTextEllipsisedCompat                                                                                                                                                                        Text.cpp         138  0x5555557d80fe 
11  gfx_draw_string_centred_clipped                                                                                                                                                                 Text.cpp         196  0x5555557d8461 
12  widget_caption_draw                                                                                                                                                                             Widget.cpp       532  0x555555652e20 
13  widget_draw                                                                                                                                                                                     Widget.cpp       90   0x55555565169f 
14  window_draw_widgets                                                                                                                                                                             Window.cpp       652  0x555555656d12 
15  window_park_price_paint                                                                                                                                                                         Park.cpp         1308 0x5555556c1ac3 
16  window_event_paint_call                                                                                                                                                                         Window.cpp       1598 0x5555557ff36a 
17  window_draw_single                                                                                                                                                                              Window.cpp       1280 0x5555557fe698 
18  window_draw                                                                                                                                                                                     Window.cpp       1163 0x5555557fe07e 
19  <lambda(rct_window *)>::operator()(rct_window *) const                                                                                                                                          Window.cpp       2121 0x555555800e70 
20  std::_Function_handler<void(rct_window *), window_draw_all(rct_drawpixelinfo *, int16_t, int16_t, int16_t, int16_t)::<lambda(rct_window *)>>::_M_invoke(const std::_Any_data &, rct_window *&&) std_function.h   316  0x55555580519f 
... <More>                                                                                                                              
```

**Comment by willthornton** [comment]
Was able to reproduce on Windows 10, running OpenRCT2 v0.2.3-ab5f5b2 develop.

**Comment by ultin00b** [comment]
Was able to reproduce on Windows 10, running OpenRCT2 v0.2.3-ab5f5b2 develop as well.

**Comment by nuclearslurpee** [comment]
Was able to reproduce on Ubuntu 18.10, running openrct2 0.2.3-develop-0~652-201907221525-g5a6f573

**Comment by jgottula** [comment]
Also able to reproduce on Windows 8.1, running ab5f5b2.

#9633 also appears to be the exact same issue.

**Comment by IntelOrca** [comment]
Fix coming soon, see #9634

Duplicate of #9626
